tools/prime-admin/src/main/kotlin/org/ostelco/tools/prime/admin/actions/Utils.kt

```diff
@@ -25,7 +25,7 @@ fun <L, R> Either<L, R>.print() = this.fold(
             this
         },
         { right ->
```

Comment:
what is the right. thing here? Please use a variable  name that is a bit more descriptive

Reply:
This "right" side of Either<Right,Left>.
It does not have any type.
The function signature hence uses "Generics" - `fun <L, R> Either<L, R>.print()`